data/human/human missions.txt

```diff
@@ -6410,13 +6412,13 @@ mission "Care Package to South 2b"
 				`	"So... that prosthetic is very new, then?"`
 					goto new
 				`	"With that prosthetic, soon you'll have a leg up on the rest of us, eh?"`
-			`	She grins broadly at your pun. "Yep! Just here to get a float chair for the first few weeks of adjustment."`
+			`	She grins broadly at your pun. "Yep! Just here to get a float chair for the first few weeks of adjustment.`
 				goto leave
 			label sorry
-			`	She waves it off, but her face softens. "Thanks. I'm here to pick up a float chair, for the first six weeks of adjustment."`
+			`	She waves it off, but her face softens. "Thanks. I'm here to pick up a float chair, for the first six weeks of adjustment.`
 				goto leave
 			label new
-			`	She nods. "Yep, just came from getting it fitted, and gotta check in at the quartermaster's to get myself a float chair for the first six weeks of adjustment to it."`
+			`	She nods. "Yep, just came from getting it fitted, and gotta check in at the quartermaster's to get myself a float chair for the first six weeks of adjustment to it.`
```

Comment:
For the record, these are not strictly necessary, since the dialogue continues in the next paragraph. It doesn't hurt to have them, though, so I won't suggest that you change it.
<img width="666" alt="image" src="https://github.com/endless-sky/endless-sky/assets/98854112/68083ee5-ad4d-4109-a7b0-496bea4c7121">